features/api/command/run.feature

```diff
@@ -60,13 +60,13 @@ Feature: Run command
     Then the specs should all pass
 
   Scenario: Non-existing executable
-    Given a file named "bin/cli" does not exist
+    Given a file named "bin/cli-app" does not exist
```

Comment:
I'll need a separate PR for this. Aruba shouldn't even be using zsh in tests, because:

1. It hides the intention of what is being tested here (I'm still unsure, but I'm just guessing it's the path to an app, really)
2. Too much of a barrier to maintain (how many people will know the BSD_ECHO option?)
3. Doesn't work on Windows and will need to be revised (PITA)

(I commented on Gitter [here](https://gitter.im/cucumber/aruba?at=56fbed02e4a8384a1bbd47ae)